parabolDocker/Dockerfile

```diff
@@ -0,0 +1,126 @@
+    && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+    && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+    && grep " node-v$NODE_VERSION.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
+    && tar -xf "node-v$NODE_VERSION.tar.xz" \
+    && cd "node-v$NODE_VERSION" \
+    && ./configure \
+    && make -j$(getconf _NPROCESSORS_ONLN) V= \
+    && make install \
+    && apk del .build-deps-full \
+    && cd .. \
+    && rm -Rf "node-v$NODE_VERSION" \
+    && rm "node-v$NODE_VERSION.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt; \
+  fi \
+  && rm -f "node-v$NODE_VERSION-linux-$ARCH-musl.tar.xz" \
+  && apk del .build-deps \
+  # smoke tests
+  && node --version \
+  && npm --version
+
+ENV YARN_VERSION 1.22.4
+
+RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
+  && for key in \
+    6A010C5166006599AA17F08146C2130DFD2497F5 \
+  ; do \
+    gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+    gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+    gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+  done \
+  && curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
+  && curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
+  && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
+  && mkdir -p /opt \
+  && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/ \
+  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarn /usr/local/bin/yarn \
+  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg \
+  && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
+  && apk del .build-deps-yarn \
+  # smoke test
+  && yarn --version
+
+
+# Install the packages required for watchman to work properly:
+#RUN apk add --no-cache libressl-dev libcrypto1.0 libgcc libstdc++
+RUN apk add --no-cache libressl-dev libgcc libstdc++
+# Copy the watchman executable binary directly from our image:
+COPY --from=icalialabs/watchman:4-alpine3.4 /usr/local/bin/watchman* /usr/local/bin/
+
+# Create the watchman STATEDIR directory:
+RUN mkdir -p /usr/local/var/run/watchman \
+ && touch /usr/local/var/run/watchman/.not-empty
+
+# (Optional) Copy the compiled watchman documentation:
+COPY --from=icalialabs/watchman:4-alpine3.4 /usr/local/share/doc/watchman* /usr/local/share/doc/
+
+RUN apk add git openssh curl python3 make gcc g++ pixman-dev cairo-dev pango-dev jpeg-dev libc6-compat \
+  && ln -s /lib64/ld-linux-x86-64.so.2  /lib/ld-linux-x86-64.so.2
+
+COPY --chown=node:node ./ /opt/parabol/
+COPY --chown=node:node ./parabolDocker/.env_test /opt/parabol/packages/server/.env
+
+USER node
+
+# Create app directory
+WORKDIR /opt/parabol
+
+COPY ./parabolDocker/docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
Super small cosmetic nit: I'd prefer this being stored in a directory named `docker` vs. `parabolDocker` as we're most certainly in the parabol project. Including the prefix `parabol` feels redundant

Reply:
I will take care of it.